app/src/main/res/layout/fragment_onboarding.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".feature.onboarding.ui.screen.OnboardingFragment">
+
+
+    <androidx.viewpager2.widget.ViewPager2
+        android:id="@+id/view_pager"
+        android:layout_width="match_parent"
+        android:layout_height="600dp"
```

Comment:
please constraint to top and bottom , plz don't do like that 600dp

Reply:
Can I use wrap content sir?

Reply:
any problem of contraint with top and bottom and height with 0dp ??